benchmark/single-source/SetTests.swift

```diff
@@ -139,6 +139,88 @@ public let SetTests = [
     tags: [.validation, .api, .Set],
     setUpFunction: { blackHole([setP, setQ]) }),
 
+  BenchmarkInfo(
+    name: "Set.isSuperset.Empty.Int",
+    runFunction: { n in run_SetIsSupersetInt(setE, setAB, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setE, setAB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Int.Empty",
+    runFunction: { n in run_SetIsSupersetInt(setAB, setE, true, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, setE]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Int0",
+    runFunction: { n in run_SetIsSupersetInt(setAB, setCD, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, setCD]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Box0",
+    runFunction: { n in run_SetIsSupersetBox(setOAB, setOCD, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOAB, setOCD]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Int25",
+    runFunction: { n in run_SetIsSupersetInt(setB, setAB, false, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setB, setAB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Box25",
+    runFunction: { n in run_SetIsSupersetBox(setOB, setOAB, false, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOB, setOAB]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Int50",
+    runFunction: { n in run_SetIsSupersetInt(setY, setXY, false, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setY, setXY]) }),
+  BenchmarkInfo(
+    name: "Set.isSuperset.Int100",
+    runFunction: { n in run_SetIsSupersetInt(setP, setQ, true, 50 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setP, setQ]) }),
+
+  BenchmarkInfo(
+    name: "Set.isStrictSuperset.Empty.Int",
+    runFunction: { n in run_SetIsStrictSupersetInt(setE, setAB, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setE, setAB]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSuperset.Int.Empty",
+    runFunction: { n in run_SetIsStrictSupersetInt(setAB, setE, true, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, setE]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSuperset.Int0",
+    runFunction: { n in run_SetIsStrictSupersetInt(setAB, setCD, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, setCD]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSuperset.Box0",
+    runFunction: { n in run_SetIsStrictSupersetBox(setOAB, setOCD, false, 5000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOAB, setOCD]) }),
+  BenchmarkInfo(
+    name: "Set.isStrictSuperset.Int25",
+    runFunction: { n in run_SetIsStrictSupersetInt(setB, setAB, false, 50 * n) },
```

Comment:
It would make sense to swap the arguments here -- we want the `isStrictSuperset` test to succeed so that we actually exercise some containment checks.